ui/src/app/views/sign-in-flow-template/component.tsx

```diff
@@ -19,19 +20,33 @@ interface ImagesSource {
 interface PageTemplateState {
   currentStep: number;
   invitationKey: string;
+  profile: Profile;
 }
 
 @withWindowSize()
 export class SignPageTemplateReact extends React.Component<any, PageTemplateState> {
   state: PageTemplateState;
   pageImages: Array<ImagesSource>;
   headerImg = '/assets/images/logo-registration-non-signed-in.svg';
+  invitationKey: string;
 
   constructor(props: object) {
     super(props);
     this.state = {
-      currentStep: 0,
-      invitationKey: ''
+      currentStep: 1,
```

Comment:
Just a note for other reviewers.(@UrsaStutsman  correct me if i am wrong) currentStep will be 0 (or rather some string as per review by @panentheos ) this is 1 right now because of testing by ursa.